src/main/java/uk/gov/pay/directdebit/payments/model/PaymentRequestEvent.java

```diff
@@ -75,32 +106,17 @@ public void setEventDate(ZonedDateTime eventDate) {
         CHARGE_CREATED,
         TOKEN_EXCHANGED,
         DIRECT_DEBIT_DETAILS_RECEIVED,
-        PAYER_CREATED;
+        PAYER_CREATED,
```

Comment:
Don't we want to add `MANDATE` as a possible Event Type? That was originally the plan, I reckon.

Reply:
Mandate is an Event type...

Reply:
Not according to the gods of source code :godmode: